packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js

```diff
@@ -68,16 +70,40 @@ export class CategoryPage extends PureComponent {
         isMatchingInfoFilter: false,
         totalPages: 1,
         is_anchor: true,
-        search: ''
+        search: '',
+        defaultPlpType: '',
+        plpTypes: []
     };
 
+    state = {};
```

Comment:
Maybe set default state here? You are setting it in the `componentDidUpdate` method which is called basically right away.

Reply:
How can you put value from prop to state here?

Reply:
`state={key:value}`?

Reply:
I mean, how to get value from props here correctly

Reply:
Of right, there's a prop value. Sorry, didn't notice it...
If that's the case I don't see an issue.